src/Marshaller.php

```diff
@@ -307,7 +307,7 @@ protected function _validate($data, $options, $isNew)
         }
 
         if ($options['validate'] === true) {
-            $options['validate'] = $this->index->validator('default');
+            $options['validate'] = $this->index->getValidator('default');
         }
         if (is_string($options['validate'])) {
             $options['validate'] = $this->index->validator($options['validate']);
```

Comment:
Need update too (getValidator)

Reply:
yes, well the ones i changed now are the ones i used and checked.
Dunno if i should just go and change them everywhere

Reply:
That would be ideal